File: DESIGN-CSharp/Templete/Adventurer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Templete
{
    public abstract class Adventurer
    {
        public int level { get; set; }
        public string type { get; set; }
    }

    public class GundamJustice : Adventurer 
    {
        public GundamJustice() 
        {
            base.type = "Gundam-Justice";
            base.level = 100;
        }
    }

    public class Sabar : Adventurer
    {
        public Sabar()
        {
            base.type = "Sabar";
            base.level = 10;
        }
    }
}

File: BehavioralPatterns/Template/Adventurer.py
class Adventurer():
    def __init__(self):
        self.level = 0
        self.type = ""

class GundamJustice(Adventurer):
    def __init__(self):
        super().__init__()
        self.type = "Gundam-Justice"
        self.level = 100

class Sabar(Adventurer):
    def __init__(self):
        super().__init__()
        self.type = "Sabar"
        self.level = 10